src/features/donations/components/treeDonation/PaymentFunctions.ts

```diff
@@ -12,10 +12,10 @@ export async function createDonation(data: any,session:any) {
         : 'en'
     }`
   }
-  if(session){
+  if(token && token !== ''){
```

Comment:
same as above - do we need to check `&& token !== ''`?

Reply:
Sometimes '' is not considered as Boolean false.